doc/nrf/device_guides/working_with_nrf/nrf70/developing/constrained.rst

```diff
@@ -29,6 +29,15 @@ You can identify and select the appropriate host MCU depending on the IoT use ca
 .. note::
    The asterisk * indicates that the SoC is supported, but the use case is not recommended due to performance or memory constraints.
 
+Firmware patch optimizations
```

Comment:
```suggestion
nRF70 Series firmware patch optimizations
```

Reply:
This was intention this page is part of the nRF70 device guides, so, no need to repeat `nRF70`

![image](https://github.com/nrfconnect/sdk-nrf/assets/100136718/ffe2cf06-587d-4330-a27f-a182373888be)

Reply:
Seem my comment on the other occasion. We need to be more verbose when there is a risk of confusion on what firmware we refer to (host or RPU), because nRF70 Series is never a standalone platform (especially in nRF Connect SDK)